torchtext/experimental/datasets/question_answer.py

```diff
@@ -110,7 +108,8 @@ def SQuAD1(*args, **kwargs):
             could also choose any one of them, for example ('train', 'test') or
             just a string 'train'. If 'train' is not in the tuple or string, a vocab
             object should be provided which will be used to process valid and/or test
-            data.
+            data. Note: we follow the convention of Stanford Question Answering datasets
```

Comment:
I'd add why this matters, e.g. we use 'dev' which is commonly referred to 'test'.

Reply:
Let's not add this for now, until we resolved this.

Reply:
Removed